test/parallel_api/experimental/for_loop.pass.cpp

```diff
@@ -296,8 +296,9 @@ test_for_loop()
         Sequence<T> in_out(n, Gen<T>());
         Sequence<T> expected = in_out;
 
-        invoke_on_all_policies<>()(test_for_loop_impl(), in_out.begin(), in_out.end(), expected.begin(), expected.end(),
-                                   in_out.size());
+        // for_loop staff is implemented for the host policies only
```

Comment:
Are only formatting changes in tests?

Reply:
added comment proposed by @MikeDvorskiy  + formatting of the next line